OrbitCore/Tcp.cpp

```diff
@@ -9,95 +9,74 @@
-    m_Connection = nullptr;
+    connection_ = nullptr;
   }
 }
 
-//-----------------------------------------------------------------------------
 void tcp_server::RegisterConnection(
-    std::shared_ptr<TcpConnection> a_Connection) {
+    std::shared_ptr<TcpConnection> connection) {
   PRINT_FUNC;
-  m_Connection = a_Connection;
+  connection_ = connection;
 }
 
-//-----------------------------------------------------------------------------
 void tcp_server::start_accept() {
   PRINT_FUNC;
 
   // creates a socket
-  TcpConnection::pointer new_connection =
-      TcpConnection::create(m_Acceptor.get_io_service());
+  auto new_connection = TcpConnection::create(acceptor_.get_io_service());
 
   // initiates an asynchronous accept operation
   // to wait for a new connection.
-  m_Acceptor.async_accept(*new_connection->GetSocket().m_Socket,
-                          std::bind(&tcp_server::handle_accept, this,
-                                    new_connection, std::placeholders::_1));
+  acceptor_.async_accept(*new_connection->GetSocket().m_Socket,
+                         std::bind(&tcp_server::handle_accept, this,
+                                   new_connection, std::placeholders::_1));
 }
 
 // handle_accept() is called when the asynchronous accept operation
 // initiated by start_accept() finishes. It services the client request
-//-----------------------------------------------------------------------------
-void tcp_server::handle_accept(TcpConnection::pointer new_connection,
+void tcp_server::handle_accept(std::shared_ptr<TcpConnection> new_connection,
                                const asio::error_code& error) {
   PRINT_FUNC;
   if (!error) {
     PRINT_FUNC;
-    m_ConnectionsSet.insert(new_connection);
+    connections_set_.insert(new_connection);
     new_connection->start();
   }
 
   start_accept();
 }
 
-//-----------------------------------------------------------------------------
-TcpConnection::~TcpConnection() {
-  PRINT_VAR("TcpConnection::~TcpConnection()");
-}
-
-//-----------------------------------------------------------------------------
 void TcpConnection::ReadMessage() {
   PRINT_FUNC;
-  asio::async_read(m_Socket, asio::buffer(&m_Message, sizeof(Message)),
-
+  asio::async_read(socket_, asio::buffer(&message_, sizeof(Message)),
                    [this](asio::error_code ec, std::size_t /*length*/) {
                      if (!ec) {
-                       m_NumBytesReceived += sizeof(Message);
+                       num_bytes_received_ += sizeof(Message);
                        ReadPayload();
                      } else {
                        PRINT_VAR(ec.message());
-                       m_Socket.close();
+                       socket_.close();
                      }
                    }
 
   );
 }
 
-//-----------------------------------------------------------------------------
 void TcpConnection::ReadWebsocketMessage() {
   /*asio::async_read( m_Socket, asio::buffer( &m_WebSocketBuffer,
```

Comment:
We're keeping this commented code for now?

Reply:
I guess there is no point, removed it.

Reply:
The websocket code used to work but was disabled when porting form Windows to Linux as it was no longer being used and I wanted to cut a dependency.  We can get rid of that code and revive it properly if we do need websockets again.

Reply:
In that case we can also remove 
char m_WebSocketBuffer[MAX_WS_HEADER_LENGTH]

Reply:
removed this.